webapp/portlet/src/main/webapp/vue-apps/profile-header/components/ProfileHeaderAvatar.vue

```diff
@@ -96,7 +96,18 @@ export default {
               this.$emit('input', uploadId);
             }
           })
-          .then(() => this.$emit('refresh'))
+          .then(() => {this.$emit('refresh');
+            if (navigator.serviceWorker && window.caches) {
```

Comment:
```suggestion
            if (navigator.serviceWorker && window.caches) {
```
This is just a workaround and we shouldn't manage service worker lifecycle inside apps

To make a better fix, you can delete Service Worker on 6.2.x, in fact, it's not used anymore. To do it, you can, as a start, just change default value of :
https://github.com/Meeds-io/gatein-portal/blob/811013bc654cd35f7100049d8f0275862be0bc35/web/portal/src/main/webapp/WEB-INF/conf/pwa/pwa-configuration.xml#L11

to make it `false` instead of `true`

Reply:
should i delete the workaround or not?